configs/config-combinations/ubuntu16_04.yaml

```diff
@@ -0,0 +1,3 @@
+bazel: "0.21.0"
```

Comment:
The service has one dep_specs.yaml which includes 1 bazel container and >=1 exec container. iiuc, it will write to multiple files in config-combinations folder, for each bazel + exec container?

Reply:
I think I'm going to change the service to accept exactly 1 bazel container and 1 toolchain container. There will be a service instance watching/updating the YAML file for each toolchain container

Reply:
cool.